packages/devtools/lib/src/memory/memory.dart

```diff
@@ -234,25 +476,63 @@ class MemoryScreen extends Screen with SetStateMixin {
     }
   }
 
+  List<String> _knownSnapshotClasses = [];
+  List<String> getAllClassesKnown() {
```

Comment:
nit: rename to `getKnownSnapshotClasses` to be consistent with `_knownSnapshotClasses`

Reply:
Done